bigbluebutton-html5/private/locales/pt_BR.json

```diff
@@ -1,97 +1,15 @@
 {
-  "app.home.greeting": "Bem-vindo {0}! Sua aprensentação começará em breve...",
-  "app.userlist.usersTitle": "Users",
-  "app.userlist.participantsTitle": "Participantes",
-  "app.userlist.messagesTitle": "Mensagens",
```

Comment:
Same in this file, I can see that some of the translated strings are deleted between 1 and 30.